### ISSUE TITLE
touchscreen controls

### ISSUE THREAD

**Issue by DennisDray** [martpie/museeks#465]
the volume control button needs to be tapped to show volume slider
which mutes sound
so need to press again to unmute
moreover, once the slider appears, the control buttons stay gone

best is to put the slider in place of the mute button

**Comment by martpie** [comment]
Never thought about touchscreen, thank your for the feedback.

I’ll try to think about a solution for that.

**Comment by dreaddr** [comment]
Yeah, this would be nice, a player that works on both mouse and touchscreen mode.
